oarepo_ui/facets.py

```diff
@@ -1,7 +1,38 @@
+from flask_login import current_user
+
 from oarepo_ui.constants import no_translation
 from oarepo_ui.utils import get_oarepo_attr, partial_format
 
+# COMMON FACETS
+
+def term_facet(field, order='desc', size=100, missing=None):
+    ret = {
+        'terms': {
+            'field': field,
+            'size': size,
+            "order": {"_count": order}
+        },
+    }
+    if missing is not None:
+        ret['terms']['missing'] = missing
+    return ret
+
+
+# ROLE FACETS
+class RoleFacets:
+    def __init__(self, **kwargs):
+        self.data = kwargs
 
+    def current_role(self):
```

Comment:
Co ostatni role uzivatele krom anonymous/authenticated

Reply:
imho se to planovalo tak, ze ostatni role se resi v podedene classe - protoze clovek muze byt ve vic rolich ale z hlediska facet pro tohle musis vybrat jednu ....

Za me ok, mergnete + release? Dik